shelter/utils/obtaintarget.go

```diff
@@ -0,0 +1,142 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"bytes"
```

Comment:
Please use `go fmt` under `shelter/utils` dir, and you can see the formatted code like this
```Go
import (
       	"bytes"
        "fmt"
	"os"
	"os/exec"
	"strings"
 )
```

Reply:
change as your comments.